cargo-dist/Cargo.toml

```diff
@@ -36,6 +36,7 @@ thiserror = "1.0.35"
 tracing = { version = "0.1.36", features = ["log"] }
 serde = { version = "1.0.144", features = ["derive"] }
 cargo_metadata = "0.15.0"
+cruet = "0.13.3"
```

Comment:
I didn't feel great about adding another dep, I just wanted to not have to think about the ClassCase conversion.